README.md

```diff
@@ -79,35 +108,65 @@ Keep track of these values for later.
 
 ### Step 3
 
-Run the command `git clone https://github.com/IBMAppModernization/web-terminal.git`, then `cd web-terminal`. First, open `chart/web-terminal/values.yaml` in a text editor of your choice. Make the following changes:
+To retrieve the source code, run the command `git clone https://github.com/IBMAppModernization/web-terminal.git`, then `cd web-terminal`. 
+
+```console
+$ git clone https://github.com/IBMAppModernization/web-terminal.git
+$ cd web-terminal
+```
+
+First, open `chart/web-terminal/values.yaml` in a text editor of your choice. Make the following changes:
 
 - `participantCount`: this should be a number, and will set up that many terminal instances at `/term1`, `/term2`, etc...
-- `tlsSecret`: the name of your TLS secret from Step 2
-- `fullDomain`: the full domain of your cluster from Step 2
-- `repository`: by default, this should be `ibmappmodernization/web-terminal` to use our Docker image, but feel free to use your own if you have another image
+- `tlsSecret`: the name of your Ingress Secret from Step 2
+- `fullDomain`: the full domain of your Ingress Subdomain from Step 2
+- `repository`: by default, this should be `ibmappmodernization/web-terminal` to use our Docker image, but feel free to use your own image in a public Docker Hub repository, if you have another image
 - `tag`: you have choices, check out the available tags [here](https://hub.docker.com/r/ibmappmodernization/web-terminal/tags)
 
 Save your file.
 
+To build your own file:
+
+```console
+$ docker login -u <docker-username> -p <password>
+$ docker build --no-cache -t web-terminal -f Dockerfile-nodejs10 .
+$ docker tag web-terminal:latest <docker-username>/web-terminal:nodejs10
```

Comment:
I think we need to add a CI job for auto-building each image and pushing to the `IBMAppModernization` namespace on Dockerhub. This would be a good spot to recommend using an “official” image for this, but i suppose people can do what they want if they have the image handy.